MC-33183: New In-store delivery feature. Updated existing information and added new topics. (#604)
magento/merchdocs: src/configuration/sales/sales-emails.md

|Send Credit Memo Comment Email Copy To|Store View|Specifies the email address of anyone to receive a copy of a credit memo comment email. Separate multiple addresses with a comma.|
|Send Credit Memo Comments Email Copy Method|Store View|Indicates the email method used to send the copy. Options include: <br/>**Bcc** - Sends a blind courtesy copy by including the recipient in the header of the same email  that is sent to the customer. The BCC recipient is not visible to the customer. <br/>**Separate Email** - Sends the copy as a separate email.|

## Order Ready For Pickup in Store

![]({% link images/images/Order Ready For Pickup in Store.png %}){: .zoom}
[_Order Ready For Pickup in Store_]({% link shipping/shipping-in-store-delivery.md %})

|Field|[Scope]({% link configuration/scope.md %})|Description|
|--- |--- |--- |

> |Enabled|Store View|When enabled, sends a transactional email for each Order Ready For Pickup in Store. Options include: Yes / No|

Reviewer: ```suggestion
|Enabled|Store View|When enabled, sends a transactional email for each order that is ready for pickup in store. Options: Yes / No|
```

Author: done, thanks!

> |Enabled|Store View|When enabled, sends a transactional email for each Order Ready For Pickup in Store. Options include: Yes / No|

Author: thanks!

> |Order Ready For Pickup Email Sender|Store View|Identifies the store contact that appears as the message sender. Default sender: General Contact|

Reviewer: ```suggestion
|Order Ready For Pickup Email Sender|Store View|Identifies the store contact that appears as the message sender. Default sender: `General Contact`|
```

Author: done!

> |Order Ready For Pickup Email Template|Store View|Identifies the template that is sent when an Order Ready For Pickup in Store. Default template: Order is Ready for Pickup|

Reviewer: ```suggestion
|Order Ready For Pickup Email Template|Store View|Identifies the template that is used for the transactional email for each order that is ready for pickup in store. Default template: `Order is Ready for Pickup`|
```

Author: done, thanks you 2!

> |Order Ready For Pickup Email Template for Guest|Store View|Identifies the template that is sent when an Order Ready For Pickup in Store for a guest. Default template: Order is Ready for Pickup for Guest|

Reviewer: ```suggestion
|Order Ready For Pickup Email Template for Guest|Store View|Identifies the template that is used for the transactional email for each order that is ready for pickup in store for a guest. Default template: `Order is Ready for Pickup for Guest`|
```

Author: done, thanks!

> |Send Order Ready For Pickup Email Copy To|Store View|Specifies the email address of anyone to receive a copy of an Order Ready For Pickup email. Separate multiple addresses with a comma.|

Reviewer: ```suggestion
|Send Order Ready For Pickup Email Copy To|Store View|Specifies the email address of anyone to receive a copy of an _Order Ready For Pickup_ email. Separate multiple addresses with a comma.|
```

Author: thanks!

> |Send Order Ready For Pickup Email Copy Method|Store View|Indicates the email method used to send the copy. Options include: <br/>**Bcc** - Sends a blind courtesy copy by including the recipient in the header of the same email that is sent to the customer. The BCC recipient is not visible to the customer. <br/>**Separate Email** - Sends the copy as a separate email.|

Reviewer: ```suggestion
|Send Order Ready For Pickup Email Copy Method|Store View|Indicates the email method used to send the copy. Options: <br/>**Bcc** - Sends a blind courtesy copy by including the recipient in the header of the same email that is sent to the customer. The BCC recipient is not visible to the customer. <br/>**Separate Email** - Sends the copy as a separate email.|
```

Author: thanks!
